### ISSUE TITLE
Bead Sort

### ISSUE THREAD

**Issue by jainaman224** [jainaman224/Algo_Ds_Notes#2141]
- [x] C
- [x] C++
- [ ] C#
- [ ] CoffeeScript
- [ ] Dart
- [ ] Go
- [x] Java
- [ ] JS
- [ ] Kotlin
- [ ] Php
- [x] Python
- [ ] Readme
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

**Comment by anshu18107** [comment]
kindly assign me @jainaman224 , I would like to work on it with  c,c++&python

**Comment by anshu18107** [comment]
Kindly assign this issue @jainaman224 ,would like to work on the javascript solution

**Comment by jainaman224** [comment]
@anshu18107 @kmrakash @ranjal007 @darkdebo Are you working on it?

**Comment by darkdebo** [comment]
Yes , working on it.Too much online assignment and exams by college.So it's take some time.

**Comment by darkdebo** [comment]
You can check that,I fix the problems.But pr is in still review.

**Comment by ranjal007** [comment]
@jainaman224  I need to implement the code of Bead sort in Kotlin also. Please also assign it to me.

**Comment by ranjal007** [comment]
@jainaman224  I am working on readme file its almost completed. Just need 1 or 2 days more.

**Comment by jainaman224** [comment]
@darkdebo Please link this issue with PR.

**Comment by ranjal007** [comment]
@jainaman224  Please review my PR

https://github.com/jainaman224/Algo_Ds_Notes/pull/2661

**Comment by m-code12** [comment]
@jainaman224 I can do this in php. Please assign .
